src/Message/MakePaymentRequest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use stdClass;
+use Omnipay\Common\Exception\InvalidRequestException;
+
+/**
+ * Make payment against an outstanding invoice amount.
```

Comment:
This could use a bit more explanation. It's not clear if the user is being charged.